examples/stable_diffusion_v2/README.md

```diff
@@ -324,7 +325,13 @@ after setting `data_path` in `run_train_v1.sh` to your dataset path.
 
 # Stable Diffusion with ControlNet
 
-# Stable Diffusion with T2I Adapter
+ControlNet controls pretrained large diffusion models to support additional input conditions. The ControlNet learns task-specific conditions in an end-to-end way, and the learning is robust even when the training dataset is small. Large diffusion models like Stable Diffusion can be augmented with ControlNets to enable conditional inputs like canny edge maps, segmentation maps, keypoints, etc.
```

Comment:
Please add hyper-link to the controlnet readme

Reply:
It already has in line 332